src/gui/settingsdialog.cpp

```diff
@@ -199,15 +199,13 @@ SettingsDialog::SettingsDialog(ownCloudGui *gui, QWidget *parent)
     _actionGroup->setExclusive(true);
 
 
-    if (Theme::instance()->multiAccount()) {
-        _addAccountAction = new ToolButtonAction(QStringLiteral("plus-solid"), tr("Add account"), this);
-        _addAccountAction->setCheckable(false);
-        connect(_addAccountAction, &QAction::triggered, this, []{
-            // don't directly connect here, ocApp might not be defined yet
-            ocApp()->gui()->runNewAccountWizard();
-        });
-        _ui->toolBar->addAction(_addAccountAction);
-    }
+    _addAccountAction = new ToolButtonAction(QStringLiteral("plus-solid"), tr("Add account"), this);
+    _addAccountAction->setCheckable(false);
+    connect(_addAccountAction, &QAction::triggered, this, [] {
+        // don't directly connect here, ocApp might not be defined yet
+        ocApp()->gui()->runNewAccountWizard();
+    });
+    _ui->toolBar->addAction(_addAccountAction);
```

Comment:
Don't we need a check here whether an account has been added already? In that case, we don't need to display the button right away.

Reply:
I could do that, but in line \#270 (still in the constructor) all accounts are added, so it will remove that button before the dialog is shown. Yes, it's a small bit of overhead, but less logic to think about later ;-)

Reply:
I could also add a comment explaining this. So, whatever you like.

Reply:
I guess that's fine then. Thanks for the explanation.